stable/nodejs/examples/package.json

```diff
@@ -4,6 +4,7 @@
     "dependencies": {
         "end-of-stream": "^1.4.1",
         "from2": "^2.3.0",
-        "lodash": "^4.17.5"
+        "lodash": "^4.17.5",
```

Comment:
can you use a different file for the new example? something like `multipart-package.json`